templates/user/profile.tmpl

```diff
@@ -31,7 +31,7 @@
 				{{else if eq .TabName "overview"}}
 					<div id="readme_profile" class="markup">{{.ProfileReadme | Str2html}}</div>
 				{{else}}
-					{{template "explore/repo_search" .}}
+					{{template "explore/repo_search" (dict "ctxData" . "tab" "repositories")}}
```

Comment:
At least, one is wrong.

![image](https://github.com/go-gitea/gitea/assets/2114189/37c28853-3828-4944-9734-f93ec58257c4)